don't break pdfium build on m68k

FIXME: Make this set by autoconf, most of the defines in build_config.h are not actually
used anyway in pdfium...

diff --git a/external/pdfium/configs/build_config.h b/external/pdfium/configs/build_config.h
index 22084c0a0836..6fd080e644fb 100644
--- a/external/pdfium/configs/build_config.h
+++ b/external/pdfium/configs/build_config.h
@@ -118,6 +118,9 @@
 #define ARCH_CPU_ARM_FAMILY 1
 #define ARCH_CPU_ARM64 1
 #define ARCH_CPU_64_BITS 1
+#elif defined(__m68k__)
+#define ARCH_CPU_M68K 1
+#define ARCH_CPU_32_BITS 1
 #else
 #error Please add support for your architecture in build/build_config.h
 #endif

